src_py/sysfont.py

```diff
@@ -24,11 +24,12 @@
 import sys
 import warnings
 from os.path import basename, dirname, exists, join, splitext
+from typing import Dict
 
 from pygame.font import Font
 
 OpenType_extensions = frozenset((".ttf", ".ttc", ".otf"))
-Sysfonts = {}
+Sysfonts: Dict[str, str] = {}
 Sysalias = {}
```

Comment:
why is `Sysalias` not typed here?

Reply:
`mypy` does not request that a type be specified (hopefully because it can intuit that from the code).